javascript/builtins/Error.json

```diff
@@ -228,6 +228,56 @@
             }
           }
         },
+        "captureStackTrace": {
+          "__compat": {
```

Comment:
MDN page under construction in https://github.com/mdn/content/pull/38048 - I assume this will auto-link once it exists.